apps/astarte_appengine_api/lib/astarte_appengine_api/device/device.ex

```diff
@@ -101,6 +131,12 @@ defmodule Astarte.AppEngine.API.Device do
     |> Enum.into(%{})
   end
 
+  defp merge_metadata(old_metadata, new_metadata) when is_map(old_metadata) do
```

Comment:
do we have a similar function for aliases?

Reply:
Yes. The two functions are now condensed in a more general `merge_data`